tests/test_main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from unittest import mock
+
+import pytest
+
+from app.main import outdated_products
+
+
+@pytest.fixture()
+def products_template():
+    yield [{"name": "salmon",
+            "expiration_date": 5,
```

Comment:
Here, for example, must be datetime, not integer.

Reply:
Ok, thanks. I figured out the issue and rewrote my tests using 'datetime' objects.